src/parser/core/metric.ts

```diff
@@ -0,0 +1,28 @@
+import { AnyEvent } from './Events';
+import Ability from './modules/Ability';
+
+export interface Info {
+  playerId: number;
+  abilities: Ability[];
+  fightStart: number;
+  fightEnd: number;
+}
```

Comment:
This seems good to start and we should also be able to easily add things to this later - I can see us wanting encounter information as well

Reply:
I'm unsure about all the fight props so far, seems like `Info` is going to keep on growing like this and that's not great. Might switch this around a bit later. I considered making our own `Fight` type that's basically a cleaned up version of WCL's response.

Reply:
including sub-types like `Fight` might be the way, yeah. I don't think `Info` needs to grow a lot, but there is metadata that needs to be communicated somehow, and `Info` is probably the vehicle for that.

Reply:
I am hesitant to provide all data that we have like report and fight in info. Some things are duplicated or slightly different. Current approach is to add things as they're needed and then refactor once it becomes clear what is needed as a whole. So far I've been adding more and more fight props (in the other PRs), so that will probably need to be added one way or another.

Reply:
Will work on that later.